alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -185,6 +179,12 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
 
         monitorResult = monitorResult.copy(inputResults = InputRunResults(listOf(inputRunResults)))
 
+        queries.forEach {
```

Comment:
Add a small comment of the purpose of this and what you are getting from `inputRunResults`